src/transaction/trustline.js

```diff
@@ -8,8 +8,7 @@ import type {Instructions, Prepare} from './types.js';
 import type {TrustLineSpecification} from '../ledger/trustlines-types.js';
 
 function convertQuality(quality) {
-  return quality === undefined ? undefined :
```

Comment:
It is always checked before
https://github.com/ripple/ripple-lib/blob/develop/src/transaction/trustline.js#L30-L34